plugins/woocommerce-admin/client/payments-welcome/style.scss

```diff
@@ -109,6 +109,19 @@
 					margin-left: 10px;
 				}
 			}
+
+			.help-section-simple {
+				font-size: 16px;
+				padding: 18px 16px;
+				span {
+					vertical-align: 6px;
+					margin-left: 10px;
+				}
+				a {
+					vertical-align: 6px;
+					margin-left: 10px;
```

Comment:
_Nitpick_: We could also use `$gap` here.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/34581/commits/579b1d63f1d8fdb76ca4449af6d23d873e438f09.

Reply:
oh, and https://github.com/woocommerce/woocommerce/pull/34581/commits/b9268a1e5ec482393d5d127f230187d22dde2e10